roles/reset/tasks/main.yml

```diff
@@ -330,6 +330,7 @@
     - /usr/libexec/kubernetes
     - /etc/origin/openvswitch
     - /etc/origin/ovn
+    - /etc/crictl.yaml
```

Comment:
This part seems good for me.
But I am not sure the reason of the other part according to the pull request description.

Reply:
sorry, I submitted some redundant modifications together. 
now it's ready